doc/config.rst

```diff
@@ -877,6 +877,18 @@ that use Sphinx's HTMLWriter class.
    named :file:`default.css` will overwrite the theme's
    :file:`default.css`.
 
+   .. note::
+
+      For security reason, dotfiles under ``html_static_path`` will
```

Comment:
typo: `reason` → `reasons`

Reply:
Thanks. Your comment is very helpful to me :-)

Reply:
Thanks for all your work on Sphinx!